packages/react-components/src/date-input/src/DateInput.tsx

```diff
@@ -43,7 +43,7 @@ export interface DatePreset {
     date: Date;
 }
 
-export interface InnerDateInputProps extends InteractionStatesProps, AriaLabelingProps {
+export interface InnerDateInputProps extends InteractionStatesProps, AriaLabelingProps, Omit<ComponentProps<"input">, "autoFocus" | "defaultValue" | "max" | "min" | "value"> {
```

Comment:
I thought replacing the "autoFocusDelay" prop for an hybrid "autoFocus" prop accepting a bool or a delay was smart... now I understand how bad of an idea it is ;)